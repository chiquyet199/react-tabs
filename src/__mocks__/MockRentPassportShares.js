export const data = {
  agencies: [
    {
      id: "j2734392-63e1-4e91-b39c-72b0a3c5e216",
      name: "testAgency1",
      type: "BUILD_TO_RENT",
      phone: "41561321548",
      address: {
        line1: "Address line 1",
        line2: "Address line 2",
        town: "town",
      },
      branches: [
        {
          id: "j1734392-63e1-4e91-b39c-72b0a3c5e216",
          agencyId: "j2734392-63e1-4e91-b39c-72b0a3c5e216",
          name: "branchTestName1",
          address: {},
          phone: "654512315",
          shareStatus: "PENDING",
          properties: [
            {
              id: "d2734392-63e1-4e91-b39c-72b0a3c5e200",
              presentation: {
                propertyType: "APARTMENT",
                bedroomsCount: 1,
                bathroomsCount: 2,
                furnishedType: "NOT_FURNISHED",
                hasParking: false,
                hasOutsideSpace: false,
                petsAllowed: true,
                smokingAllowed: true,
              },
              propertyImages: [
                {
                  id: "d2734392-63e1",
                  property: {},
                  url:
                    "https://thespaces.com/wp-content/uploads/2017/02/paris-apartments-for-rent-Rue-de-Varenne-III-3-1.jpg",
                  isPrimary: true,
                  createdAt: 1540551441,
                  updatedAt: 1540551441,
                },
              ],
              rentFrequency: "MONTHLY",
              rentAmount: 99,
              landlord: {},
              address: {
                line1: "Address line 1",
                line2: "Address line 2",
                town: "town",
                state: "state",
                postCode: "postCode",
                countryCode: "countryCode",
              },
              createdBy: "h2734392-63e1-4e91-b39c-72b0a3c5e216",
              agencyId: "j2734392-63e1-4e91-b39c-72b0a3c5e216",
              branchId: "j1734392-63e1-4e91-b39c-72b0a3c5e216",
            },
          ],
        },
        {
          id: "c1734392-63e1-4e91-b39c-72b0a3c5e216",
          agencyId: "j2734392-63e1-4e91-b39c-72b0a3c5e216",
          name: "branchTestName2",
          address: {},
          phone: "654512315",
          shareStatus: "SHARED",
          properties: [
            {
              id: "d2734392-63e1-4e91-b39c-72b0a3c5e201",
              presentation: {
                propertyType: "APARTMENT",
                bedroomsCount: 1,
                bathroomsCount: 2,
                furnishedType: "NOT_FURNISHED",
                hasParking: false,
                hasOutsideSpace: false,
                petsAllowed: true,
                smokingAllowed: true,
              },
              propertyImages: [
                {
                  id: "d2734392-63e1",
                  property: {},
                  url:
                    "https://thespaces.com/wp-content/uploads/2017/02/paris-apartments-for-rent-Rue-de-Varenne-III-3-1.jpg",
                  isPrimary: true,
                  createdAt: 1540551441,
                  updatedAt: 1540551441,
                },
              ],
              rentFrequency: "MONTHLY",
              rentAmount: 99,
              landlord: {},
              address: {
                line1: "Address line 1",
                line2: "Address line 2",
                town: "town",
                state: "state",
                postCode: "postCode",
                countryCode: "countryCode",
              },
              createdBy: "h2734392-63e1-4e91-b39c-72b0a3c5e216",
              agencyId: "j2734392-63e1-4e91-b39c-72b0a3c5e216",
              branchId: "c1734392-63e1-4e91-b39c-72b0a3c5e216",
            },
          ],
        },
      ],
    },
    {
      id: "a2734392-63e1-4e91-b39c-72b0a3c5e216",
      name: "testAgency1",
      type: "BUILD_TO_RENT",
      phone: "41561321548",
      address: {
        line1: "Address line 1",
        line2: "Address line 2",
        town: "town",
      },
      branches: [
        {
          id: "a1734392-63e1-4e91-b39c-72b0a3c5e216",
          agencyId: "a2734392-63e1-4e91-b39c-72b0a3c5e216",
          name: "branchTestName1",
          address: {},
          phone: "654512315",
          shareStatus: "PENDING",
          properties: [
            {
              id: "d2734392-63e1-4e91-b39c-72b0a3c5e200",
              presentation: {
                propertyType: "APARTMENT",
                bedroomsCount: 1,
                bathroomsCount: 2,
                furnishedType: "NOT_FURNISHED",
                hasParking: false,
                hasOutsideSpace: false,
                petsAllowed: true,
                smokingAllowed: true,
              },
              propertyImages: [
                {
                  id: "d2734392-63e1",
                  property: {},
                  url:
                    "https://thespaces.com/wp-content/uploads/2017/02/paris-apartments-for-rent-Rue-de-Varenne-III-3-1.jpg",
                  isPrimary: true,
                  createdAt: 1540551441,
                  updatedAt: 1540551441,
                },
              ],
              rentFrequency: "MONTHLY",
              rentAmount: 99,
              landlord: {},
              address: {
                line1: "Address line 1",
                line2: "Address line 2",
                town: "town",
                state: "state",
                postCode: "postCode",
                countryCode: "countryCode",
              },
              createdBy: "h2734392-63e1-4e91-b39c-72b0a3c5e216",
              agencyId: "a2734392-63e1-4e91-b39c-72b0a3c5e216",
              branchId: "a1734392-63e1-4e91-b39c-72b0a3c5e216",
            },
          ],
        },
        {
          id: "dd734392-63e1-4e91-b39c-72b0a3c5e216",
          agencyId: "a2734392-63e1-4e91-b39c-72b0a3c5e216",
          name: "branchTestName2",
          address: {},
          phone: "654512315",
          shareStatus: "PENDING",
          properties: [
            {
              id: "d2734392-63e1-4e91-b39c-72b0a3c5e201",
              presentation: {
                propertyType: "APARTMENT",
                bedroomsCount: 1,
                bathroomsCount: 2,
                furnishedType: "NOT_FURNISHED",
                hasParking: false,
                hasOutsideSpace: false,
                petsAllowed: true,
                smokingAllowed: true,
              },
              propertyImages: [
                {
                  id: "d2734392-63e1",
                  property: {},
                  url:
                    "https://thespaces.com/wp-content/uploads/2017/02/paris-apartments-for-rent-Rue-de-Varenne-III-3-1.jpg",
                  isPrimary: true,
                  createdAt: 1540551441,
                  updatedAt: 1540551441,
                },
              ],
              rentFrequency: "MONTHLY",
              rentAmount: 99,
              landlord: {},
              address: {
                line1: "Address line 1",
                line2: "Address line 2",
                town: "town",
                state: "state",
                postCode: "postCode",
                countryCode: "countryCode",
              },
              createdBy: "h2734392-63e1-4e91-b39c-72b0a3c5e216",
              agencyId: "a2734392-63e1-4e91-b39c-72b0a3c5e216",
              branchId: "dd734392-63e1-4e91-b39c-72b0a3c5e216",
            },
          ],
        },
      ],
    },
  ],
};
