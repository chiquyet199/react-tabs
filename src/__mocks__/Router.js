import React from "react";
import Routing from "../utils/routing";

const { Router } = Routing; // eslint-disable-line
const history = {
  location: "/",
  listen: jest.fn(),
  createHref: jest.fn(),
  replace: jest.fn(),
  push: jest.fn(),
};

const TestRouter = ({children}) => (
  <Router history={history}>{children}</Router>
);

export default TestRouter;
