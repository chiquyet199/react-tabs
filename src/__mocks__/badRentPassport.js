export default {
  userId: "67e4b887-466e-4abe-8d12-bc87204e5df8",
  status: "SUBMITTED",
  aboutInfo: {
    email: "jonathan@gmail.com",
    firstName: "Jonathan",
    lastName: "Beaumont",
    middleName: null,
    gender: "MALE",
    dayOfBirth: 1,
    monthOfBirth: 1,
    yearOfBirth: 1970,
    maritalStatus: "MARRIED",
    phoneNumber: "01123456789",
    nationality: "UK",
    isSmoker: true,
    hasPets: true,
    hasChildren: true,
    status: "GREEN",
    messages: [],
  },
  annualIncomeInfo: {
    salary: 100000,
    pension: 0,
    interest: 0,
    benefits: 0,
    rentals: 0,
    other: 0,
    grossAnnualIncome: 100000,
    sourcesOfIncome: ["Employment"],
    status: "GREEN",
    messages: [],
  },
  financialInfo: {
    bankrupt: false,
    creditScore: 790,
    depositFree: true,
    maximumEligible: 1800.266,
    activeCCJs: 1450,
    activeCCJYear: 2016,
    activeCCJMonth: 1,
    activeCCJValue: 0,
    previousCCJs: 0,
    previousCCJsYear: 0,
    previousCCJsMonth: 0,
    activeInsolvencies: 3250,
    activeInsolvenciesYear: 2016,
    activeInsolvenciesMonth: 11,
    previousInsolvencies: false,
    previousInsolvenciesYear: 0,
    previousInsolvenciesMonth: 0,
    status: "YELLOW",
    messages: [
      "credit.insolvency.failed",
      "credit.countyCourtJudgments.failed",
    ],
  },
  legalInfo: {
    pep: true,
    mortality: "failed",
    legalWarnings: true,
    sanctions: true,
    status: "RED",
    messages: [
      "watchList.politicallyExposedPerson.failed",
      "watchList.legalAndRegulatoryWarnings.failed",
      "watchList.sanction.failed",
      "identity.mortality.failed",
    ],
  },
  workInfo: [
    {
      workStatus: "EMPLOYED",
      companyName: "companyName",
      jobTitle: "jobTitle",
      income: 100000,
      employmentTime: "FULL_TIME",
      employmentType: "CONTRACT",
      workedSinceMonth: 12,
      workedSinceYear: 2017,
      studentType: "POST_GRADUATE_STUDENT",
      reference: {
        contactFirstName: "Silvester",
        contactLastName: "Stallone",
        contactEmailAddress: "rambo666@gmail.com",
        referredAt: "12-12-2016",
      },
      status: "YELLOW",
      messages: ["requestReference"],
    },
  ],
  residenceInfo: [
    {
      line1: "Addresss line 1",
      town: "Town",
      state: "State",
      postCode: "UL11223",
      countryCode: "UK",
      residenceType: "RENTING_FROM_AGENT",
      livedSinceMonth: 1,
      livedSinceYear: 2016,
      livedToMonth: null,
      livedToYear: null,
      reference: {
        contactFirstName: "Jonathan",
        contactLastName: "Beaumont",
        contactEmailAddress: "jonathan@gmail.com",
        referredAt: "01-01-2018",
      },
      status: "GREEN",
      messages: [],
    },
  ],
};
