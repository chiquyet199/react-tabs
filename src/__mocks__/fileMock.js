export function MockFile() {}

MockFile.prototype.create = (
  name = "mock.jpg",
  size = 1024,
  mimeType = "image/png",
) => {
  function range(count) {
    let output = "";
    for (let i = 0; i < count; i += 1) {
      output += "a";
    }

    return output;
  }

  const blob = new Blob([range(size)], {type: mimeType});
  blob.lastModifiedDate = new Date();
  blob.name = name;

  return blob;
};
