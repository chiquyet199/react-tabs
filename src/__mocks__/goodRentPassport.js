export default {
  userId: "67e4b887-466e-4abe-8d12-bc87204e5df8",
  status: "SUBMITTED",
  aboutInfo: {
    email: "jonathan@gmail.com",
    firstName: "Jonathan",
    lastName: "Beaumont",
    middleName: null,
    gender: "MALE",
    dayOfBirth: 20,
    monthOfBirth: 11,
    yearOfBirth: 1970,
    maritalStatus: "MARRIED",
    phoneNumber: "01123456789",
    nationality: "UK",
    isSmoker: true,
    hasPets: true,
    hasChildren: true,
    status: "GREEN",
    messages: [],
  },
  annualIncomeInfo: {
    salary: 100000,
    pension: 0,
    interest: 0,
    benefits: 0,
    rentals: 0,
    other: 0,
    grossAnnualIncome: 100000,
    sourcesOfIncome: ["Employment"],
    status: "YELLOW",
    messages: [],
  },
  financialInfo: {
    bankrupt: false,
    creditScore: 930,
    depositFree: true,
    maximumEligible: 1800.266,
    activeCCJs: 0,
    activeCCJYear: 0,
    activeCCJMonth: 0,
    activeCCJValue: 0,
    previousCCJs: 0,
    previousCCJsYear: 0,
    previousCCJsMonth: 0,
    activeInsolvencies: 0,
    activeInsolvenciesYear: 0,
    activeInsolvenciesMonth: 0,
    previousInsolvencies: false,
    previousInsolvenciesYear: 0,
    previousInsolvenciesMonth: 0,
    status: "GREEN",
    messages: [],
  },
  legalInfo: {
    pep: false,
    mortality: "passed",
    legalWarnings: false,
    sanctions: false,
    status: "GREEN",
    messages: [],
  },
  workInfo: [
    {
      workStatus: "EMPLOYED",
      companyName: "companyName",
      jobTitle: "jobTitle",
      income: 100000,
      employmentTime: "FULL_TIME",
      employmentType: "CONTRACT",
      workedSinceMonth: 12,
      workedSinceYear: 2017,
      studentType: "POST_GRADUATE_STUDENT",
      reference: {
        contactFirstName: "Silvester",
        contactLastName: "Stallone",
        contactEmailAddress: "rambo666@gmail.com",
        referredAt: "12-12-2016",
      },
      status: "GREEN",
      messages: [],
    },
  ],
  residenceInfo: [
    {
      line1: "Addresss line 1",
      town: "Town",
      state: "State",
      postCode: "UL11223",
      countryCode: "UK",
      residenceType: "RENTING_FROM_AGENT",
      livedSinceMonth: 1,
      livedSinceYear: 2016,
      livedToMonth: null,
      livedToYear: null,
      reference: {
        contactFirstName: "Jonathan",
        contactLastName: "Beaumont",
        contactEmailAddress: "jonathan@gmail.com",
        referredAt: "01-01-2018",
      },
      status: "GREEN",
      messages: [],
    },
  ],
};
