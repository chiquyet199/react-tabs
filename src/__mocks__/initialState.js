export const initialState = {
  locale: {
    currentLocale: "en-US",
    translations: {
      common: {
        postcode_lookup: {
          postcode: "Postcode",
          find_address: "Find address",
          address_not_listed: "Address not listed",
        },
      },
      home__welcome: "Welcome to RNW starter app",
      home__loggedin: "You are now logged in",
      login__email__label: "Email address",
      login__password__label: "Password",
      login__email__placeholder: "example@example.com",
      login__password__placeholder: "******",
      login__continue__with__facebook: "Continue with Facebook",
      login: "Sign in",
      create__account: "Create account",
      login__forgot__password: "Forgot password",
      logout: "Logout",
      back: "Back",
      submit: "Submit",
      rent_passport: "Rent Passport",
      deposit_free: "Deposit Free",
      home_management: "Home Management",
      authorize: "Authorize",
      terms_header: "Canopy terms & conditions",
      terms:
        "1. INTRODUCTION\n\nBy signing up for an account with Canopy, you are agreeing to be bound by the following terms and conditions (“Terms of Service”). These Terms of Service are to be read in conjunction with:\n\nour Privacy Policy, which sets out the terms on which we process any personal data we collect from you, or that you or others provide to us; and\n\nour Cookies Policy, which sets out information about the cookies we use.\n\nThese Terms of Service are important. Please read them carefully as they contain details of our statutory and regulatory responsibilities to you, and your contractual obligations to us. They also set out the basis on which we provide, and you are permitted to use, the various services made available on the website www.findyourcanopy.com and the mobile application known as ‘Canopy’ (together, the “Platform”), including the following:",
      verify__email__title__text: "Verify your email address",
      verify__email__text:
        "We need to know you are you before creating a RentPassport. Please click the link in the email we sent you.",
      open__email__client: "Open my email client",
      resend__verification__email: "Resend verification email",
      not__received__an__email: "Not received an email?",
      forgot_password: "Forgot your password?",
      reset_password: "Change your password",
      tenant_reference_submitted: "Reference submitted",
      tenant_reference_thanks_for_time:
        "Thanks for taking the time to submit a tenant reference for",
      tenant_reference_did_you_know:
        "Did you know that Canopy offers the following property management tools to tenants, landlords and agents:",
      tenant_reference_rent_passport_text:
        "Full tenant referencing and property management including deposits and payments",
      tenant_reference_deposit_free_text:
        "Qualified tenants can pay a premium to cover their rental security deposit",
      tenant_reference_home_management_text:
        "With a valid UK Open Banking account, tenants are able to have rent payments contribute to their Experian credit report",
      find_out_more: "Find out more",
      tenant_reference_title: "Tenant reference request",
      upload_tenant_reference_document: "Upload tenant reference document",
      upload_tenant_reference_document_description:
        "If you have a manual form you usually use, please upload it here.",
      upload: "Upload",
      submit_reference: "Submit reference",
      tenant_reference_request: "Tenant reference request",
      complete_reference_form_description:
        " tenant of yours, has authorised us to request a reference from you.\n" +
        "As you know a landlord reference is significant information when determining eligibility of a prospective tenant, so please consider the questions below carefully and feel free to make additional comments.",
      upload_reference_document: "Upload reference document",
      complete_reference_form: "Complete reference form",
      complete_reference_form_a: ", a ",
      truelayer: {
        truelayer_auth_title: "Truelayer Authorize",
      },
      employer_reference_form_description:
        " has advised us that you are their employer and has authorised us to request a reference from you.\n" +
        "As you know an employment reference is significant information when determining eligibility of a prospective tenant, so please consider the questions below carefully and feel free to make additional comments.",
    },
  },
  auth: {
    user: null,
    isLoggedIn: false,
  },
  dimensions: {
    fontScale: 1,
    height: 901,
    scale: 1,
    width: 1920,
    size: "L",
  },
  inputField: {
    showPassword: true,
  },
  form: {},
  partialForm: {
    values: {},
  },
  progress: 0,
  notification: {
    type: null,
    message: "",
  },
  user: {
    profile: {
      firstName: "FirstName",
      lastName: "LastName",
    },
  },
  router: {
    location: {
      pathname: "/tenant-reference-request/complete",
      search: "",
      hash: "",
    },
    action: "POP",
  },
  lookups: {
    addresses: [],
  },
  topbar: {
    haveTopbar: true,
    topbarBtn: {
      haveTopbarButton: false,
      topbarButtonText: "",
      btnType: "transparent_green",
      topbarButtonIcon: undefined,
      hideBackButton: false,
    },
    topbarCenter: {
      topbarCenterText: "Center",
    },
    topbarLeft: {
      topbarLeftText: "Back",
    },
  },
  images: {
    accepted: [],
  },
  rentPassport: {
    aboutYou: {
      email: "jonathan@gmail.com",
      firstName: "Jonathan",
      lastName: "Beaumont",
      middleName: null,
      gender: "MALE",
      dayOfBirth: 20,
      monthOfBirth: 11,
      yearOfBirth: 1970,
      maritalStatus: "MARRIED",
      phoneNumber: "01123456789",
      nationality: "UK",
      isSmoker: true,
      hasPets: true,
      hasChildren: true,
      status: "GREEN",
      messages: [],
    },
    financial: {
      salary: 100000,
      pension: 0,
      interest: 0,
      benefits: 0,
      rentals: 0,
      other: 0,
      grossAnnualIncome: 100000,
      sourcesOfIncome: ["Employment"],
      status: "YELLOW",
      messages: [],
      bankrupt: false,
      creditScore: 930,
      depositFree: true,
      maximumEligible: 1800.266,
      activeCCJs: 0,
      activeCCJYear: 0,
      activeCCJMonth: 0,
      activeCCJValue: 0,
      previousCCJs: 0,
      previousCCJsYear: 0,
      previousCCJsMonth: 0,
      activeInsolvencies: 0,
      activeInsolvenciesYear: 0,
      activeInsolvenciesMonth: 0,
      previousInsolvencies: false,
      previousInsolvenciesYear: 0,
      previousInsolvenciesMonth: 0,
    },
    rentPassportLegal: {
      pep: false,
      mortality: "passed",
      legalWarnings: false,
      sanctions: false,
      status: "GREEN",
      messages: [],
    },
    whatYouDo: {
      workInfo: [
        {
          workStatus: "EMPLOYED",
          companyName: "companyName",
          jobTitle: "jobTitle",
          income: 100000,
          employmentTime: "FULL_TIME",
          employmentType: "CONTRACT",
          workedSinceMonth: 12,
          workedSinceYear: 2017,
          studentType: "POST_GRADUATE_STUDENT",
          reference: {
            contactFirstName: "Silvester",
            contactLastName: "Stallone",
            contactEmailAddress: "rambo666@gmail.com",
            referredAt: "12-12-2016",
          },
          status: "GREEN",
          messages: [],
        },
      ],
    },
    whereYouLived: {
      residenceInfo: [
        {
          line1: "Addresss line 1",
          town: "Town",
          state: "State",
          postCode: "UL11223",
          countryCode: "UK",
          residenceType: "RENTING_FROM_AGENT",
          livedSinceMonth: 1,
          livedSinceYear: 2016,
          livedToMonth: null,
          livedToYear: null,
          reference: {
            contactFirstName: "Jonathan",
            contactLastName: "Beaumont",
            contactEmailAddress: "jonathan@gmail.com",
            referredAt: "01-01-2018",
          },
          status: "GREEN",
          messages: [],
        },
      ],
    },
  },
};
