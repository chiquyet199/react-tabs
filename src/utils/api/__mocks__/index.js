import {injectMocks, extractScenarioFromLocation} from "data-mocks";
import example from "./test-mocks";
import users from "./users";

let scenario;

const {NODE_ENV} = process.env;

if (NODE_ENV !== "test") {
  scenario = extractScenarioFromLocation(window.location);
}

injectMocks(
  {
    default: [...example, ...users.default],
    failedSignup: [...users.failedSignup],
    takenEmail: [...users.takenEmail],
    failedToSendResetToken: [...users.failedToSendResetToken],
  },
  scenario,
);
