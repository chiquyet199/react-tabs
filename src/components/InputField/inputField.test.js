import React from "react";
import {shallow} from "enzyme";
import InputFieldComponent from "./inputField";
import configureStore from "redux-mock-store";
import {initialState} from "../../__mocks__/initialState";
import {extend} from "underscore";

const getMockProps = () => ({
  onChangeText: jest.fn(),
  showPassword: jest.fn(),
  label: "label",
  password: false,
  shownPassword: false,
  errorText: "",
  defaultValue: "",
  textColor: "#6b7c93",
  tintColor: "#6b7c93",
  keyboardType: null,
  multiline: false,
  autoCapitalize: "sentences",
  prefix: "",
  theme: {
    colors: {
      alertRed: "red",
    },
  },
});
const mockStore = configureStore(getMockProps());

const getComponent = props => {
  const parsedProps = extend(getMockProps(), props);

  return shallow(<InputFieldComponent {...parsedProps} />, {
    context: {store: mockStore(initialState)},
  });
};

describe("<InputField/> component", () => {
  it("check snapshot", () => {
    expect(getComponent()).toMatchSnapshot();
  });
});
