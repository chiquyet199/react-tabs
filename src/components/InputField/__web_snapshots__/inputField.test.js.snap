// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputField/> component check snapshot 1`] = `
<Connect(WithTheme(InputField))
  autoCapitalize="sentences"
  defaultValue=""
  errorText=""
  keyboardType={null}
  label="label"
  multiline={false}
  onChangeText={[MockFunction]}
  password={false}
  prefix=""
  showPassword={[Function]}
  shownPassword={true}
  textColor="#6b7c93"
  theme={
    Object {
      "colors": Object {
        "alertRed": "red",
      },
    }
  }
  tintColor="#6b7c93"
/>
`;
